packages/suite/src/components/wallet/AmountComponent.tsx

```diff
@@ -0,0 +1,44 @@
+import { isNftTokenTransfer, formatAmount, getTxOperation } from '@suite-common/wallet-utils';
+import { TokenTransfer } from '@trezor/connect';
+
+import { FormattedNftAmount } from 'src/components/suite/FormattedNftAmount';
+import { FormattedCryptoAmount } from 'src/components/suite/FormattedCryptoAmount';
+
+type AmountComponentProps = {
+    transfer: TokenTransfer;
+    withLink?: boolean;
+    withSign?: boolean;
+    alignMultitoken?: 'flex-end' | 'flex-start';
+};
+
+export const AmountComponent = ({
+    transfer,
+    withLink = false,
+    withSign = false,
+    alignMultitoken,
+}: AmountComponentProps): React.ReactNode => {
+    const operation = getTxOperation(transfer.type);
```

Comment:
this can be inside the `if (withSign) {` condition

Reply:
No because it's also used before the condition in if(isNftTokenTransfer(transfer))